dynamics-gp/financials/receivables-management-part3-transaction-activity.md

```diff
@@ -496,7 +496,7 @@ Use the Customer/Vendor Consolidations window to create consolidation documents
 
 1. Open the Customer/Vendor Consolidations window. (Sales \>\> Transactions \>\> Customer/Vendor Trx.)
 
-    ![91622f2488ad897b2334f1007d0d7e36.jpg](media/91622f2488ad897b2334f1007d0d7e36.jpg)
+    ![Screenshot of Customer/Vendor Consolidations window, showing default entries and empty input boxes.](media/91622f2488ad897b2334f1007d0d7e36.jpg)
```

Comment:
Use "a" or "the" before the word "Customer" for this and all screenshots in the PR.